indicator/src/main/java/com/fuzz/indicator/StateProxy.java

```diff
@@ -56,6 +58,18 @@
     void resendPositionInfo(CutoutViewIndicator cvi, float assumedIndicatorPosition);
 
     /**
+     * In some use cases, it may be handy to have a reference to the original
+     * View when binding a representation thereof. See
+     * {@link LayeredViewGenerator#onBindChild(View, CutoutViewLayoutParams, View)}
+     * for details.
+     *
+     * @param cviPosition    position of a child within the CutoutViewIndicator
+     * @return the view which directly inspired the indicator's child
+     */
+    @Nullable
+    View getOriginalViewFor(int cviPosition);
```

Comment:
Discuss: should this really be part of the `StateProxy` interface? Or maybe there should a separate, `OriginalViewFinder`-like interface that StateProxies can optionally implement, and can be set separately?